mozfldp/simulation_util.py

```diff
@@ -127,35 +116,15 @@ def server_update(
                 all_classes,
             )
 
-            client_coefs = append(client_coefs, coefs)
-
-            client_intercepts = append(client_intercepts, intercept)
-
-            num_samples.append(len(client_feature))
-
-        # calculate the new server weights based on new weights coming from client
-        new_coefs = np.zeros(init_weight[0].shape, dtype=np.float64, order="C")
-        new_intercept = np.zeros(init_weight[1].shape, dtype=np.float64, order="C")
-
-        for index, user_id in enumerate(user_ids):
-            client_coef = client_coefs[index]
-            client_intercept = client_intercepts[index]
-
-            n_k = num_samples[index]
-            added_coef = [value * (n_k) / sum(num_samples) for value in client_coef]
-            added_intercept = [
-                value * (n_k) / sum(num_samples) for value in client_intercept
-            ]
+            serv.send_weights(coefs, intercept, num_features)
```

Comment:
This would now by handled as the final step of `Client.update_and_submit_weights`